ynput/OpenPype
# AYON: Change of server url should work as expected

## Changelog Description
Using login action in tray menu to change server url should correctly start new process without issues of missing bundle or previous url.

## Additional info
Just pop values of previous session from environment variables before restart.

## Testing notes:
1. Open ayon launcher
2. Login to server 1
3. In tray hit Login action
4. Change login to server 2
5. That should restart tray and correctly use new login and bundle should be determined to prod, staging or dev.
